python/lsst/meas/base/compensatedGaussian/_compensatedTophat.py

```diff
@@ -44,7 +44,7 @@ class SingleFrameCompensatedTophatFluxConfig(SingleFramePluginConfig):
         doc="The aperture radii (in pixels) to measure the top-hats.",
         dtype=int,
         minLength=1,
-        default=[12,],
+        default=[12, ],
```

Comment:
spurious change

Reply:
That was needed so that `scons` would complete (linting issue).